source/isaaclab/isaaclab/sensors/tacsl_sensor/visuotactile_sensor_cfg.py

```diff
@@ -0,0 +1,185 @@
+    """Filename of the background image within the data directory."""
+
+    calib_path: str = "polycalib.npz"
+    """Filename of the polynomial calibration data within the data directory."""
+
+    real_background: str = "real_bg.npy"
+    """Filename of the real background data within the data directory."""
+
+    image_height: int = 320
+    """Height of the tactile image in pixels."""
+
+    image_width: int = 240
+    """Width of the tactile image in pixels."""
+
+    num_bins: int = 120
+    """Number of bins for gradient magnitude and direction quantization."""
+
+    mm_per_pixel: float = 0.0877
+    """Millimeters per pixel conversion factor for the tactile sensor."""
+
+
+##
+# Visuo-Tactile Sensor Configuration
+##
+
+
+@configclass
+class VisuoTactileSensorCfg(SensorBaseCfg):
+    """Configuration for the visuo-tactile sensor.
+
+    This sensor provides both camera-based tactile sensing and force field tactile sensing.
+    It can capture tactile RGB/depth images and compute penalty-based contact forces.
+    """
+
+    class_type: type = VisuoTactileSensor
+
+    # Sensor type and capabilities
+    render_cfg: GelSightRenderCfg = GelSightRenderCfg()
+    """Configuration for GelSight sensor rendering.
+
+    This defines the rendering parameters for converting depth maps to realistic tactile images.
+    Defaults to GelSight R1.5 parameters. Use predefined configs like GELSIGHT_R15_CFG or
+    GELSIGHT_MINI_CFG from isaaclab_assets.sensors for standard sensor models.
+    """
+
+    enable_camera_tactile: bool = True
+    """Whether to enable camera-based tactile sensing."""
+
+    enable_force_field: bool = True
+    """Whether to enable force field tactile sensing."""
+
+    # Force field configuration
+    num_tactile_rows: int = 20
+    """Number of rows of tactile points for force field sensing."""
+
+    num_tactile_cols: int = 25
+    """Number of columns of tactile points for force field sensing."""
+
+    tactile_margin: float = 0.003
+    """Margin for tactile point generation (in meters)."""
+
+    contact_object_prim_path_expr: str | None = None
+    """Prim path expression to find the contact object for force field computation.
+
+    This specifies the object that will make contact with the tactile sensor. The sensor will automatically
+    find the SDF collision mesh within this object for optimal force field computation.
+
+    .. note::
+        The expression can contain the environment namespace regex ``{ENV_REGEX_NS}`` which
+        will be replaced with the environment namespace.
+
+        Example: ``{ENV_REGEX_NS}/ContactObject`` will be replaced with ``/World/envs/env_.*/ContactObject``.
+
+    .. attention::
+        For force field computation to work properly, the contact object must have an SDF collision mesh.
+        The sensor will search for the first SDF mesh within the specified prim hierarchy.
+    """
+
+    # Force field physics parameters
+    tactile_kn: float = 1.0
```

Comment:
Recommend using more understandable names like `normal_contact_stiffness` to make it clear from the variable name